content/post/predictions-entre-haine-et-passion.md

```diff
@@ -0,0 +1,69 @@
+
+Après extraction des données, nous avons découverts des chiffres intéressants. Nous souhaitons les partager ici, car on y trouve certaines similitudes et différences entre bibliothèques.
+
+<!--more-->
+
+[Pour rappel](/rero-ils-s-expose-aux-tests), nous développons {{< smallcaps "rero ils" >}} avec l'aide de bibliothèques pilotes, qui testent le système et nous fournissent un retour d'expérience de la part de professionnel·le·s du terrain. Nos six bibliothèques pilotes sont les suivantes:
+
+* [Bibliothèque cantonale jurassienne](https://www.jura.ch/occ/bicj)
+* [Bibliothèque de Bulle](https://musee-gruerien.ch/)
+* [Bibliothèque de la Ville de la Chaux-de-Fonds](http://cdf-bibliotheques.ne.ch/)
+* [Bibliothèque municipale de Delémont](http://www.delemont.ch/fr/Tourisme-culture-et-loisirs/Vie-culturelles/Bibliotheque/Bibliotheque.html)
+* [Bibliothèque publique et universitaire de Neuchâtel](http://bpun.unine.ch/)
+* [Médiathèque Valais](https://www.mediatheque.ch/) (et ses 4 sites de Brigue, Martigny, St-Maurice, Sion)
+
+## Où est-on le plus friand de prédictions?
+
+Nous sommes tombé·e·s sur ces chiffres plus ou moins par hasard; nous ne voulons donc accuser personne... mais on peut tout de même constater que les Valaisans ont une tendance plus forte à vouloir contrôler l'entrée des fascicules 🤭
+
+
+| Bibliothèque pilote                                         | Nombre de holdings | Nombre de holdings avec prédiction | Ratio |
+| ----------------------------------------------------------- | ------------------ | ---------------------------------- | ----- |
+| Médiathèque Valais (Sion)                                   | 17710              | 3892                               | 22%   |
+| Bibliothèque cantonale jurassienne                          | 6788               | 993                                | 15%   |
+| Médiathèque Valais (St-Maurice)                             | 1218               | 141                                | 12%   |
+| Médiathèque Valais (Brigue)                                 | 2209               | 210                                | 10%   |
+| Bibliothèque de Bulle                                       | 2411               | 223                                | 9%    |
+| Bibliothèque publique et universitaire de<br>Neuchâtel      | 18761              | 1672                               | 9%    |
+| Médiathèque Valais (Martigny)                               | 2333               | 196                                | 8%    |
+| Bibliothèque de la Ville de La Chaux-de-<br>Fonds (sans BJ) | 7021               | 435                                | 6%    |
+| Bibliothèque municipale de Delémont (adultes et jeunes)     | 1587               | 102                                | 6%    |
+
```

Comment:
You can add `{{< bootstrap-table "table table-striped table-bordered" >}}` just before the table, and `{{< /bootstrap-table >}}`, after, and it will improved the html renderind, like in this post: https://rero21.ch/un-pas-vers-l-evolution-du-modele-bibliographique-rero-ils/

I need to document what is possible with our blog... Sorry.